sdk/servicebus/Azure.Messaging.ServiceBus/tests/Message/MessageLiveTests.cs

```diff
@@ -249,6 +251,78 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
             }
         }
 
+        private static readonly object[] s_amqpValues =
+        {
+            "string",
+            5,
+            long.MaxValue,
+            (byte) 1,
+            3.1415926,
+            new decimal(3.1415926),
+            DateTimeOffset.Parse("3/24/21").UtcDateTime,
+            'c',
+            new Guid("55f239a6-5d50-4f6d-8f84-deed326e4554"),
+            new List<string>{"first", "second"},
+            new int[] {1, 2, 3},
+            new AmqpSymbol("symbol"),
```

Comment:
@KrzysztofCwalina this test case highlights a potential issue that we are introducing - With the addition of the AmqpValue/AmqpSequence support, customers could directly send data using the Microsoft.Azure.Amqp type AmqpSymbol. This means that we would need to always depend on this library within Service Bus.

Reply:
Would we?   If we changed to a different AMQP library, customers would have to directly reference `Microsoft.Azure.Amqp` themselves to use the type.   If they did and passed in `AmqpSymbol`, we would package it as any other object type, no?    Shouldn't we be using that same behavior now - not treating any type received as a body element with any special significance?

Reply:
The other way of phrasing this would be:  "how can we ensure that we don't assign any special meaning to different objects passed in as part of the body?"

Reply:
>we would package it as any other object type, no? 

AFAIK symbol is an AMQP primitive and is packaged in a special way. http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-types-v1.0-os.html#section-primitive-type-definitions 1.6.21 symbol

Reply:
Conceptually, yes.   What I was poking at was that we not recognize this specific type as a special `AmqpSymbol` for these purposes,  just as a generic object instance to avoid unintentional dependencies on this specific AMQP lib.

Reply:
The issue arises when this type is used with Service Bus, and thus Microsoft AMQP. This library has special knowledge of AmqpSymbol so it is able to serialize it correctly. The concern is that if we wanted to switch serialization libraries we could break people. Though I would argue that we will always need to at least leave a dependency in for Microsoft Amqp in case someone is using one of the public types via the transitive dependency. Because of this, we could figure out a way to still serialize the AmqpSymbol type using a new hypothetical AMQP library.